### DISCUSSION TITLE
Change Language

### DISCUSSION THREAD

**Discussion by Walid-Abdeljawed** [edupyter/EDUPYTER39#5] [Q&A] [answered]
Hi,
how to change language of jupyter lab help from english to french

**Comment by laurentabbal** [comment, marked as answer]
Hi,

* Open the Edupyter 'Console'
* Execute `pip install jupyterlab-language-pack-fr-fr`
* Restart JupyterLab
* Go to 'settings > Language' and choose French

I will add the French package in the next release.

**Comment by Walid-Abdeljawed** [comment]
Hi
Thank you very much

Le mer. 12 janv. 2022 à 13:05, Laurent Abbal ***@***.***> a
écrit :
